tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -204,11 +217,20 @@
 
 #define DEV_NULL_STR    0
 
+#ifndef CORE
 #define READ_KERN(ptr) ({ typeof(ptr) _val;                             \
                           __builtin_memset(&_val, 0, sizeof(_val));     \
                           bpf_probe_read(&_val, sizeof(_val), &ptr);    \
                           _val;                                         \
                         })
+#else
+// Try using READ_KERN here, just don't embed them in each other
```

Comment:
As we previously discussed, maybe we can change this macro to get the arguments in a comma separated list (which is also supported by the BPF_CORE_READ and BPF_PROBE_READ macros given by libbpf)

Reply:
Agreed, this requires a little rewiring but I can do it.

Reply:
I'll take a crack at it for this PR